src/helloworld/interfaces/hello_controller.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 type (
-	// HelloController represents our first simple controller
+	// Hello represents our first simple controller
 	HelloController struct {
 		responder *web.Responder
 	}
```

Comment:
might be an excessive comment refactoring

Reply:
Thanks - resolved :-)